cmd/skaffold/app/cmd/render.go

```diff
@@ -40,8 +44,12 @@ func NewCmdRender() *cobra.Command {
 		WithDescription("[alpha] Perform all image builds, and output rendered Kubernetes manifests").
 		WithCommonFlags().
 		WithFlags(func(f *pflag.FlagSet) {
-			f.BoolVar(&showBuild, "loud", false, "Show the build logs and output")
+			f.VarP(&fromBuildOutputFile, "build-artifacts", "a", "Don't build, but use file created with `skaffold build --file-output`")
+			f.BoolVar(&offline, "offline", false, `Do not connect to Kubernetes server API for manifest creation and validation.
+This is helpful when no Kubernetes cluster is available (e.g. GitOps model).
+No metadata.namespace attribute is injected in this case - the manifest content does not get changed.`)
 			f.StringVar(&renderOutputPath, "output", "", "file to write rendered manifests to")
+			f.BoolVar(&showBuild, "loud", false, "Show the build logs and output")
```

Comment:
This flag seems ambiguous and non-obvious - is it feasible to move into a separate PR for its own discussion? 

I feel like this should be called `--show-build-logs` or `--exit-after-logs`?

Reply:
The `--loud` flag was there before. I only reordered the arguments alphabetically (not really, though 🤦‍♂️, sorry). Out of scope for this review. I'll change the order back so the diff looks sane.